### ISSUE TITLE
How to add dropout layer in new RNN API

### ISSUE THREAD

**Issue by lilhope** [apache/incubator-mxnet#5130]
I tried this code but failed:
```
import mxnet as mx
a = mx.sym.Variable('data')
embed = mx.symbol.Embedding(data=a,input_dim=200,output_dim=20,name='embed')
stack = mx.rnn.SequentialRNNCell()
for i in range(2):
      LSTMCell = mx.rnn.LSTMCell(num_hidden=20,prefix='lstm_l%d_'%i)
      dropucell = mx.rnn.DropoutCell(LSTMCell,0.5,0.5)
      stack.add(dropucell)
outputs,states = mx.rnn.rnn_unroll(stack,10,inputs=embed)
```
Thanks in advance!

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
